prereqs.adoc

```diff
@@ -22,26 +22,36 @@ on your machine.
 
 == AWS IAM Permissions
 
-If you already have an AWS Account, you need to create an IAM user and attach these http://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies.html[IAM policies].
+If you already have an AWS Account, you need to create an IAM user here: https://console.aws.amazon.com/iam/home?#/users$new
+Just give it a useful name (e.g. `workshop`) and make sure you check the `Programmatic access` box:
 
-    AmazonEC2FullAccess
-    AmazonRoute53FullAccess
-    AmazonS3FullAccess
-    IAMFullAccess
-    AmazonVPCFullAccess
+image::images/add-iam-user.png[]
```

Comment:
Can you also add the CLI version of assigning these perms?

Reply:
What do you mean by that?

Reply:
These permissions can also be specified using AWS CLI. Can you include that as well?

Reply:
you're talking about these cli calls, right?
https://github.com/kubernetes/kops/blob/master/docs/aws.md#setup-iam-user

Reply:
maybe we should push all of the images to the setup-iam-user page and just keep the original page to point to it

Reply:
Want to make sure the perms are captured and shown using AWS CLI as that is a preferred approach by a lot of developers.

Reply:
ok - will update it now. Will basically copy/paste the ones from kops and adjust names, etc

Reply:
LMK when PR is updated